### DISCUSSION TITLE
Microsoft Remote Desktop Beta from macos to windows hang about a day, it will make the vscode editor can't type anything

### DISCUSSION THREAD

**Discussion by AiHaibara** [microsoft/vscode-discussions#1297] [Extension Development QnA] [answered]
system macos 14.5 windows 11
vscode 1.89.1
close the Microsoft Remote Desktop Beta or not seems both will make the vscode editor can't type anything even after reconnect to the windows.
when i close the vscode, and open it for new it will recovery

**Comment by isidorn** [comment, marked as answer]
This is a forum for VS Code extension developers. Please ask this question on stack overflow
